src/popup/components/feedback-editor.tsx

```diff
@@ -12,13 +15,105 @@ const styleMap = {
   },
 }
 
-export function FeedbackEditor({ editorState, updateEditorState }: FeedbackEditorProps): JSX.Element {
+const LaunchIcon = (): JSX.Element => {
   return (
-    <Editor
-      placeholder="What's your feedback?"
-      editorState={editorState}
-      onChange={editorState => updateEditorState({ editorState })}
-      customStyleMap={styleMap}
-    />
+    <svg viewBox="0 0 24 24" className="link-tooltip__launch-icon">
+      <path d="M19 19H5V5h7V3H5a2 2 0 00-2 2v14a2 2 0 002 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z" />
+    </svg>
   )
 }
+
+const TwitterLink = ({ handle }: { handle: string }): JSX.Element => {
+  const url = `https://twitter.com/${handle}`
```

Comment:
```typescript
  const url = `twitter.com/${handle.slice(1)}`
  const withProtocol = `https://${url}`
  return (
    <a className="link-tooltip__anchor" href={withProtocol} target="_blank" rel="noopener noreferrer">
      {url}
      <LaunchIcon />
    </a>
  )
```

This way so that the link is `twitter.com/abc` not `twitter.com/@abc` and it appears without the protocol to the user